parts/configure-swarm-cluster.sh

```diff
@@ -159,18 +161,18 @@ echo "$HOSTADDR $VMNAME" | sudo tee -a /etc/hosts
 
 echo "Installing and configuring docker"
 
+# simple general command retry function
+retrycmd_if_failure() { for i in 1 2 3 4 5; do $@; [ $? -eq 0  ] && break || sleep 5; done ; }

```

Comment:
@anhowe See this function and the below invocations to address your retry apt commands suggestion.